src/Browscap/Data/Validator/PropertiesValidator.php

```diff
@@ -0,0 +1,91 @@
+<?php
+declare(strict_types = 1);
+namespace Browscap\Data\Validator;
+
+class PropertiesValidator implements ValidatorInterface
+{
+    /**
+     * validates the fully expanded properties
+     *
+     * @param array  $properties Data to validate
+     * @param string $key
+     *
+     * @throws \LogicException
+     */
+    public function validate(array $properties, string $key) : void
+    {
+        if (!array_key_exists('Version', $properties)) {
```

Comment:
Can just use assertions here to simplify

Reply:
changed